src/test/java/api/requests/RequestsAPIMoveTests.java

```diff
@@ -152,56 +170,6 @@ public void canMoveAShelfHoldRequestToAnAvailableItem()
     assertThat(uponInterestingTimesQueue.getTotalRecords(), is(1));
   }
 
-  @Test
-  public void cannotMoveARecallRequestToAnEmptyQueue()
-    throws InterruptedException,
-    ExecutionException,
-    TimeoutException,
-    MalformedURLException {
-  
-    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
-    IndividualResource uponInterestingTimes = itemsFixture.basedUponInterestingTimes();
-    
-    IndividualResource james = usersFixture.james();
-    IndividualResource jessica = usersFixture.jessica();
-    IndividualResource charlotte = usersFixture.charlotte();
-
-    // james checks out basedUponSmallAngryPlanet
-    loansFixture.checkOutByBarcode(smallAngryPlanet, james);
-
-    // charlotte checks out basedUponSmallAngryPlanet
-    loansFixture.checkOutByBarcode(uponInterestingTimes, charlotte);
-
-    // make recall requests for smallAngryPlanet
-    IndividualResource requestByJessica = requestsFixture.placeRecallRequest(
-      smallAngryPlanet, jessica, new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC));
-
-    // move jessica's recall request from smallAngryPlanet to uponInterestingTimes
-    Response response = requestsFixture.attemptMove(new MoveRequestBuilder(
-      requestByJessica.getId(),
-      uponInterestingTimes.getId(),
-      null
-    ));
-
-    assertThat("Move request should have correct response status code", response.getStatusCode(), is(422));
-    assertThat("Move request should have correct response message",
-      response.getJson().getJsonArray("errors").getJsonObject(0).getString("message"),
-      is("Recalls can't be moved to checked out items that have not been previously recalled."));
-
-    requestByJessica = requestsClient.get(requestByJessica);
-    assertThat(requestByJessica.getJson().getString(REQUEST_TYPE), is(RequestType.RECALL.getValue()));
-    assertThat(requestByJessica.getJson().getInteger("position"), is(1));
-    assertThat(requestByJessica.getJson().getString("itemId"), is(smallAngryPlanet.getId().toString()));
-    retainsStoredSummaries(requestByJessica);
-
-    // check item queues are correct size
-    MultipleRecords<JsonObject> smallAngryPlanetQueue = requestsFixture.getQueueFor(smallAngryPlanet);
-    assertThat(smallAngryPlanetQueue.getTotalRecords(), is(1));
-    
-    MultipleRecords<JsonObject> uponInterestingTimesQueue = requestsFixture.getQueueFor(uponInterestingTimes);
-    assertThat(uponInterestingTimesQueue.getTotalRecords(), is(0));
-  }
-
   @Test
   public void canMoveARecallRequest()
```

Comment:
Is this when there is an empty request queue for the destination item, or in general?